src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1730,7 +1731,7 @@ because xbuild doesn't support framework reference assemblies.
 		DestinationFiles="@(_AndroidResolvedSatellitePaths->'$(MonoAndroidLinkerInputDir)%(DestinationSubDirectory)%(Filename)%(Extension)')"
 		SkipUnchangedFiles="true"
 	/>
-	<Touch Files="@(ResolvedUserAssemblies->'$(MonoAndroidLinkerInputDir)%(Filename)%(Extension)')" />
```

Comment:
@dellis1972: Is this change correct? IIRC in Debug builds, `@(ResolvedAssemblies)` may contain *system-installed* assemblies such as `Mono.Android.dll`, which we almost certainly don't want to be `<Touch/>`ing...

Reply:
What made me think this was a mistake was the previous line:
```xml
<Copy
	SourceFiles="@(ResolvedAssemblies)"
	DestinationFiles="@(ResolvedAssemblies->'$(MonoAndroidLinkerInputDir)%(Filename)%(Extension)')"
	SkipUnchangedFiles="true" />
```
I was thinking `DestinationFiles` from `<Copy />` needs to match `Files` from `<Touch />`.

If an item in `@(ResolvedAssemblies)` was the system `Mono.Android.dll`, for example, `@(ResolvedUserAssemblies->'$(MonoAndroidLinkerInputDir)%(Filename)%(Extension)')` should evaluate to `obj\Debug\linksrc\Mono.Android.dll`.

Reply:
I think @jonathanpeppers is right on this one. We are touching the files in the Intermediate dir so it should be ok.